src/components/Home.react.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Home = () => {
```

Comment:
Is there a reason we use this syntax instead of writing it as a `class extends React Component` or `React.createClass`